After update to pytest-asyncio new errors shows in retelimit tests

on a v2.15.0.27rc4:
```
testsuite/tests/apicast/policy/rate_limit/test_rate_limit_connection.py:247: in client
    async with application.api_client() as client:
               ^^^^^^^^^^^^^^^^^^^^^^^^
../virtualenvs/3scale-py-testsuite-8_Y7lf5G/lib/python3.14/site-packages/httpx/client.py:2016: in _aexit
    await self.transport._aexit(exc_type, exc_value, traceback)
../virtualenvs/3scale-py-testsuite-8_Y7lf5G/lib/python3.14/site-packages/httpx/transports/default.py:372: in _aexit
    await self.pool._aexit(exc_type, exc_value, traceback)
../virtualenvs/3scale-py-testsuite-8_Y7lf5G/lib/python3.14/site-packages/httpcore/async/connection_pool.py:364: in _aexit
    await self.aclose()
../virtualenvs/3scale-py-testsuite-8_Y7lf5G/lib/python3.14/site-packages/httpcore/_async/connection_pool.py:353: in aclose
    await self._close_connections(closing_connections)
../virtualenvs/3scale-py-testsuite-8_Y7lf5G/lib/python3.14/site-packages/httpcore/_async/connection_pool.py:345: in _close_connections
    await connection.aclose()
../virtualenvs/3scale-py-testsuite-8_Y7lf5G/lib/python3.14/site-packages/httpcore/_async/connection.py:173: in aclose
    await self._connection.aclose()
../virtualenvs/3scale-py-testsuite-8_Y7lf5G/lib/python3.14/site-packages/httpcore/_async/http11.py:258: in aclose
    await self._network_stream.aclose()
../virtualenvs/3scale-py-testsuite-8_Y7lf5G/lib/python3.14/site-packages/httpcore/_backends/anyio.py:53: in aclose
    await self._stream.aclose()
../virtualenvs/3scale-py-testsuite-8_Y7lf5G/lib/python3.14/site-packages/anyio/streams/tls.py:248: in aclose
    await self.transport_stream.aclose()
../virtualenvs/3scale-py-testsuite-8_Y7lf5G/lib/python3.14/site-packages/anyio/_backends/_asyncio.py:1397: in aclose
    self._transport.close()
/usr/lib64/python3.14/asyncio/selector_events.py:1216: in close
    super().close()
/usr/lib64/python3.14/asyncio/selector_events.py:869: in close
    self._loop.call_soon(self._call_connection_lost, None)
/usr/lib64/python3.14/asyncio/base_events.py:827: in call_soon
    self._check_closed()
/usr/lib64/python3.14/asyncio/base_events.py:550: in _check_closed
    raise RuntimeError('Event loop is closed')
E   RuntimeError: Event loop is closed
```

--- a/pytest.ini
+++ b/pytest.ini
@@ -25,7 +25,6 @@ junit_logging=all
 junit_family=xunit2
 junit_log_passing_tests=0
 asyncio_mode=auto
-asyncio_default_fixture_loop_scope=module
 addopts = --last-failed-no-failures=none
 
 #addopts = --pdbcls=IPython.terminal.debugger:Pdb

